io.github.antimicrox.antimicrox.yml

```diff
@@ -0,0 +1,24 @@
+app-id: io.github.antimicrox.antimicrox
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: antimicrox
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
```

Comment:
Does it work with wayland?

Reply:
Not yet. We need to develop and test it a bit to make sure it's working correctly. Does it need another review if we add Wayland access?

Reply:
Not really, just adding socket=wayland is enough.